docs/version0.18.md

```diff
@@ -0,0 +1,47 @@
+<!--
+* Copyright (c) 2017, 2019 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+
+# What's new in version 0.18.0
+
+The following new features and notable changes since v 0.17.0 are included in this release:
+
+- [jextract on macOS](#jextract-on-macos)
+- [Add more changes here...](#add-more-changes-here)
+
+
+## Features and changes
+
+### jextract on macOS
+
+The [`jextract` tool](tool_jextract.md) is now available on macOS&reg; platforms (as well as AIX&reg; and Linux&trade;) for all current versions of OpenJDK: 8, 11, and 13.
+
```

Comment:
I think we need to clarify the bit that is actually new, which is jextract on macos for Java 8.

Reply:
Fair point... I think the body of the text could stay as is, but perhaps change the header to:
### jextract now available on macOS for OpenJDK version 8

Reply:
Yes, that would be good - covers the new change and also the resulting overall status.

Reply:
Changes made and restaged